apps/docs/vite.config.ts

```diff
@@ -1,10 +1,11 @@
-import vue from "@vitejs/plugin-vue";
 import { fileURLToPath, URL } from "node:url";
 import { defineConfig } from "vite";
+import { VITE_BASE_CONFIG } from "./node_modules/@sit-onyx/shared/vite.config.base";
 
 // https://vitejs.dev/config
 export default defineConfig({
-  plugins: [vue()],
+  ...VITE_BASE_CONFIG,
+  plugins: [],
```

Comment:
```suggestion
```

Isn't this obsolete?

Reply:
Hi, not completely for the vitepress config we have to remove the vue plugin for it to work.
But Storybook doesn't work without the vue plugin, so I moved the overwrite to the vitepress config.